### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by tBrognoli**

### CODE REVIEW

File: luqum/tests/test_es_integration.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            ["Harry Potter and the Chamber of Secrets"],
+        )
+
+    def test_fuzzy_search(self):
+        self.assertListEqual(
+            self.ask_luqum("title:Gublet~2", self.s),
+            ["Harry Potter and the Goblet of Fire"],
+        )
+
+    def tearDown(self):
```
Comment (ahivert) [inline luqum/tests/test_es_integration.py]: Use ``tearDownClass(cls)`` -> https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDownClass

File: luqum/tests/test_es_integration.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    if ES6:
+        doc_type = "doc"
+    else:
+        doc_type = "book"
+
+    bulk(s, actions, index="bk", doc_type=doc_type, refresh=True)
+
+
+class LuqumRequestTestCase(TestCase):
+    def ask_luqum(self, req, s):
```
[the remark this answers is not in the document]
Comment (ahivert) [reply luqum/tests/test_es_integration.py]: I would put this method after setUpClass

File: luqum/tests/test_es_integration.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        self.assertListEqual(
+            self._ask_luqum("publication_date:[2005-01-01 TO 2010-12-31]"),
+            [
+                "Harry Potter and the Half-Blood Prince",
+                "The Tales of Beedle the Bard",
+                "Harry Potter and the Deathly Hallows",
+            ],
+        )
+
+    def test_int_search(self):
```
Comment (ahivert) [inline luqum/tests/test_es_integration.py]: Rename to ``test_int_range_search(self)``

Comment (ahivert) [reply luqum/tests/test_es_integration.py]: Add a test to search in like ``n_pages:500``

**Comment by alexgarel** [comment]
@tBrognoli tout est bon sauf le coverage qui a diminué !

**Comment by alexgarel** [comment]
@tBrognoli congrats !

**Pull request merged**